client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -60,6 +60,70 @@ function PlaceholderPostEdit( props ) {
 	);
 }
 
+function RelatedPostsEditItem( props ) {
+	const previewClassName = 'related-posts__preview';
+
+	return (
+		<div className={ `${ previewClassName }-post` }>
+			{ props.displayThumbnails && props.post.img && props.post.img.src && (
```

Comment:
A bit of a nitpick since we are doing a lot of props.post.
it might be better to extract that out. with
const { post } = props;

Reply:
True. This may also shorten the line a bit.